### DISCUSSION TITLE
LDAP multiple servers - AUTH_LDAP_SERVER_URI

### DISCUSSION THREAD

**Discussion by jpvrenen** [netbox-community/netbox#9710] [Q&A]
Netbox version: 3.2.6

When defining multiple LDAP servers:
**ldap_config.py**
AUTH_LDAP_SERVER_URI = "ldap://IP1 ldap://IP2"

In combination with adding a timeout statement (ldap.set_option):
**netbox/netbox/authentication.py**
 
        try:
            getattr(ldap_config, 'AUTH_LDAP_SERVER_URI')
        except AttributeError:
            raise ImproperlyConfigured(
                "Required parameter AUTH_LDAP_SERVER_URI is missing from ldap_config.py."
            )
       # Add below Timeout statement potentially check if multiple servers
        else:
            ldap.set_option(ldap.OPT_NETWORK_TIMEOUT, 5)

It will AUTH to the second IP after given timeout. Wondering if adding the ldap timeout option as a parameter (for example in ldap_config.py) could potentially be part of the distro?

**Comment by akotlubovskiy** [comment]
could you please dump your ldap_config configuration.py ? I'm trying to set up netbox authorization for 2 ldap servers, it's not working yet
